Dockerfile

```diff
@@ -59,6 +59,7 @@ RUN echo "cHJpbnRmICJrcnp5c3p0b2YuZ29yZ29sZXdza2lAZ21haWwuY29tXG41MTcyXG4gKkN2dW
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
                     fsl-core=5.0.9-1~nd+1+nd16.04+1 \
+                    fsl-atlases=5.0.7-2 \
```

Comment:
We may not need this, but we need some file named ['MNI152_T1_2mm_brain.nii.gz'](https://github.com/rhr-pruim/ICA-AROMA/blob/master/ICA_AROMA_functions.py#L146) in the ${FSLDIR}/data/standard directory. [Here](http://neuro.debian.net/pkgs/fsl-mni152-templates.html#binary-pkg-fsl-mni152-templates) is a smaller package from neurodebian that has what ICA_AROMA needs. Alternatively we could use a symlink to the MNI152NLin2009cAsym template that's named MNI152_T1_2mm_brain.nii.gz in the directory that ICA_AROMA expects. I guess I don't know how those images differ, so I don't know if that is a viable option.